tools/docker/development/docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: "3.7"
+
+services:
+    db:
+        image: "mariadb:bionic"
+        container_name: "rathena-dev_db"
+        ports:
+            - "3306:3306" # allow DB connections from host
+        volumes:
+            - "rathenadb:/var/lib/mysql" # save database to local disk
+            - "../../sql-files/:/docker-entrypoint-initdb.d" # initialize db with ./sql-files
```

Comment:
Is this a little dangerous with the tools folder in sql-files? The conversion of engine types and the MD5 password setting (if the server config isn't setup for this password type).

Unless I'm hyping this up and it doesn't crawl to sub-directories, then ignore me!

Reply:
It does map the directory to the container but i think the container only queries for sql files. The change I needed to do was because the dir is now a level below